pkgs/apps/multiwfn/default.nix

```
@@ -1,44 +1,36 @@
-{ stdenv, lib, makeWrapper, autoPatchelfHook, unzip,
-  writeScriptBin, fetchurl, glibc, xorg, libGL, motif
-} :
+{ stdenv, lib, makeWrapper, gfortran, unzip, fetchurl, xorg, libGL, motif, mkl }:
 
 stdenv.mkDerivation rec {
   pname = "multiwfn";
-  version = "3.7";
+  version = "3.8";
```

```suggestion
  version = "3.8-2022-08-24";
```
It is inconvenient, when there is no stable release/link, but we can at least tag with a date. As long as you have a cached version, you will not even notice that the link broke.

---

OK, added the date :)

> As long as you have a cached version, you will not even notice that the link broke.

That is true and cachix should help there. But only for those who actually use it. So maybe someone else notices it first when it breaks :smile: